preprocessing/config.yml

```diff
@@ -0,0 +1,34 @@
+########################################################################################################
```

Comment:
i would advise against uploading a config file for sct_run_batch-- there is hardcoded path in it-- let's just stick to the CLI, the syntax is easy enough and it is explained in the README. 

see eg how it's done in:
- https://github.com/sct-pipeline/gm-challenge#getting-started
- https://github.com/ivadomed/model_seg_ms_mp2rage#prepare-the-data

Reply:
If we are no longer using a config file, the Note at the bottom of the page of [Running the sample script...](https://spinalcordtoolbox.com/user_section/tutorials/analysis-pipelines-with-sct/sct_run_batch-usage.html) tutorial should perhaps be modified.

Reply:
@kiristern thank you for chipping in!  The note at the end of [this page](https://spinalcordtoolbox.com/user_section/tutorials/analysis-pipelines-with-sct/sct_run_batch-usage.html) refers to SCT's usage, while I was specifically referring to not using a config file for this repository only (ie: SCT users can still use a config file if they want to, but this discussion was specifically about removing the config file when running the preprocessing script to prepare the data and instead document the syntax directly in the README).